src/Microsoft.TestPlatform.Utilities/InferRunSettingsHelper.cs

```diff
@@ -204,6 +206,16 @@ public static void UpdateDesignMode(XmlDocument runSettingsDocument, bool design
         AddNodeIfNotPresent(runSettingsDocument, DesignModeNodePath, DesignModeNodeName, designModeValue);
     }
 
+    /// <summary>
+    /// Updates the <c>RunConfiguration.BatchSize</c> value for a run settings. Doesn't do anything if the value is already set.
+    /// </summary>
+    /// <param name="runSettingsDocument">Document for runsettings xml</param>
+    /// <param name="batchSizeValue">Value to set</param>
+    public static void UpdateBatchSize(XmlDocument runSettingsDocument, long batchSizeValue)
```

Comment:
Seems more like `SetBatchSize` or `AddBatchSize` because it doesn't update the value (it does nothing if already set).

Reply:
Leaving as-is as it's following pattern defined by other methods.